### ISSUE TITLE
[request] Datetime could be selected with just click the date

### ISSUE THREAD

**Issue by lzp4ever** [quasarframework/quasar#718]
It has to click the set button after picking the time, is that better to bypass it optionally?    

_Sent from my OnePlus ONEPLUS A3010 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by spectrolite** [comment]
I agree that it would be great to have an option to bypass the seemingly useless "OK" button...
But..
On mobile it is often quite hard to pick the correct date on the first try in a small grid (or with fat fingers hehe), and it's good to be able to correct a mistake before committing. Plus it seems like a universal convention, and we're trying to follow convention by default as much as possible.

So I would say, yes we need that option for some cases, and we would also probably need to be able to set differently for touch platforms vs. mouse platforms. But the default behaviour must stay, and the option should be wielded wisely.

**Comment by RichAyotte** [comment]
Here's a **_temporary_** solution.

```js
import {after} from 'meld'

mounted() {
  const cal = this.$refs.startDateCalendar
  after(cal.$refs.target, 'setDay', function() {
    cal.__update()
    cal.close()
  }.bind(cal))
}
```

Be careful here. I'm accessing a private function and code like this can be dangerous if you don't know what you're doing so document it well and include a link to this issue.  If you haven't done any AOP programming, you might not be aware of possible memory leaks or other gotchas. Use this code at your own peril. I'm sure you've written good tests so you should be fine.

Here's the AOP lib that I'm using. https://github.com/cujojs/meld Quite useful in these types of situations.

**Comment by rstoenescu** [comment]
Hi, This has been implemented in v0.15, so closing this.
